### ISSUE TITLE
Because of BLM banner change, the site became inaccessible

### ISSUE THREAD

**Issue by RAYDENFilipp** [facebook/draft-js#2494]
Because of this change, the site became inaccessible

What happened to links
![image](https://user-images.githubusercontent.com/24476415/87437697-54e35500-c5f7-11ea-9c37-e56ec94e122d.png)

It is hard to see links.

_Originally posted by @RAYDENFilipp in https://github.com/facebook/draft-js/pull/2454#issuecomment-658213125_

**Comment by RAYDENFilipp** [comment]
A probable solution might be:

1. Reduce the greyscale down to 20%-30%.

1. Remove the grayscale filter at all but leave the banner on all pages to be visible.

**Comment by claudiopro** [comment]
This is a very good bug report, the Draft.js website shouldn't use color alone to render links. This is a [common accessibility failure mode](https://www.w3.org/TR/WCAG20-TECHS/F73.html) for WCAG guidelines.

**Comment by stefan-krajnik** [comment]
You just shot yourself in the foot and now you're trying to fix it with a tape.
Links are just one part of the problem, the website is extremely difficult to read because of missing text highlighting. 

Reverting https://github.com/facebook/draft-js/commit/2a0c5bbe1709ed32b0ceedad420a81af3e84947f commit will fix the issue.

**Comment by RAYDENFilipp** [comment]
@stefan-krajnik Exactly. Fixing links won't solve the problem, the whole site causes headache when reading for more than half an hour.
